### PR TITLE
samples: bluetooth: direct_test_mode: Do not reset fem parameters

### PR DESCRIPTION
**Pull request by KAGA164**
Do not reset front-end module parameters like antenna,
gain, delay on the DTM Reset command. Testers like
Anritsu issues the DTM Reset command at every test
begging. So testing with different than default parameters
was not possible for aytomatic script testing.

### CODE REVIEW

**Review by grochu — COMMENTED** [review]
I added doc-only comments.

File: samples/bluetooth/direct_test_mode/README.rst [hunk 1]
```diff
@@ -167,6 +167,12 @@ Vendor specific commands can be divided into different categories as follows:
 * If the Length field is set to ``5`` (symbol ``FEM_ACTIVE_DELAY_SET``), the Frequency field sets the front-end module (FEM) activation delay in microseconds relative to the radio start.
   By default, this value is set to (radio ramp-up time - front-end module (FEM) TX/RX settling time).
 * All other values of Frequency and Length field are reserved.
+* If the Length field is set to ``6`` (symbol ``FEM_DEFAULT_PARAMS_SET``) and the Frequency filed is set to any value, the front-end module parameters like antenna output, gain, delay will be set to default value.
+
+.. note::
+  Front-and module configuration like antenna output, gain and active delay are not set to default values after reception of the DTM Reset command.
+  Testers like for example Anritsu MT885 issues a Reset command at every test beginning so automatic script testing would not be possible for front-end modules with parameters different than default.
+  If you change default front-end module parameters, you can restore it by sending ``FEM_DEFAULT_PARAMS_SET`` command or by power cycling your front-end module.
```
Comment (peknis) [inline samples/bluetooth/direct_test_mode/README.rst:175]:
```suggestion
  Front-end module configuration parameters, such as antenna output, gain, and active delay, are not set to their default values after the DTM reset command.
  Testers, for example Anritsu MT885, issue a reset command in the beginning of every test.
  Therefore, you cannot run automated test scripts for front-end modules with other than the default parameters.
  
  If you have changed the default parameters of the front-end module, you must restore them.
  You can either send the ``FEM_DEFAULT_PARAMS_SET`` command or power cycle the front-end module.
```

**Review by peknis — CHANGES_REQUESTED** [review]

File: samples/bluetooth/direct_test_mode/README.rst (hunk 1 above)
Comment (KAGA164) [reply samples/bluetooth/direct_test_mode/README.rst]: Fixed

File: samples/bluetooth/direct_test_mode/README.rst [hunk 2]
```diff
@@ -167,6 +167,12 @@ Vendor specific commands can be divided into different categories as follows:
 * If the Length field is set to ``5`` (symbol ``FEM_ACTIVE_DELAY_SET``), the Frequency field sets the front-end module (FEM) activation delay in microseconds relative to the radio start.
   By default, this value is set to (radio ramp-up time - front-end module (FEM) TX/RX settling time).
 * All other values of Frequency and Length field are reserved.
+* If the Length field is set to ``6`` (symbol ``FEM_DEFAULT_PARAMS_SET``) and the Frequency filed is set to any value, the front-end module parameters like antenna output, gain, delay will be set to default value.
```
[the remark this answers is not in the document]
Comment (KAGA164) [reply samples/bluetooth/direct_test_mode/README.rst]: Fixed

File: samples/bluetooth/direct_test_mode/README.rst (hunk 1 above)
[the remark this answers is not in the document]
Comment (KAGA164) [reply samples/bluetooth/direct_test_mode/README.rst]: Done

File: samples/bluetooth/direct_test_mode/README.rst (hunk 2 above)
[the remark this answers is not in the document]
Comment (KAGA164) [reply samples/bluetooth/direct_test_mode/README.rst]: Done

**Review by grochu — APPROVED** [review]

File: doc/nrf/releases/release-notes-changelog.rst [hunk 3]
```diff
@@ -84,6 +84,13 @@ For lists of protocol-specific changes, see `Protocols`_.
 Bluetooth samples
 -----------------
 
+* Updated:
+
+  * :ref:`direct_test_mode` sample:
+
+    * The Front-end module test parameters are not set to their default value after the DTM reset command.
+    * Added the ``FEM_DEFAULT_PARAMS_SET`` Vendor specific command for restoring default Front-end module parameters.
```
Comment (peknis) [inline doc/nrf/releases/release-notes-changelog.rst:92]:
```suggestion
    * Added the vendor-specific ``FEM_DEFAULT_PARAMS_SET`` command for restoring the default Front-end module parameters.
```

**Review by peknis — CHANGES_REQUESTED** [review]

File: doc/nrf/releases/release-notes-changelog.rst [hunk 4]
```diff
@@ -84,6 +84,13 @@ For lists of protocol-specific changes, see `Protocols`_.
 Bluetooth samples
 -----------------
 
+* Updated:
+
+  * :ref:`direct_test_mode` sample:
+
+    * The Front-end module test parameters are not set to their default value after the DTM reset command.
```
Comment (peknis) [inline doc/nrf/releases/release-notes-changelog.rst:91]:
```suggestion
    * The front-end module test parameters are not set to their default value after the DTM reset command.
```

**Review by peknis — CHANGES_REQUESTED** [review]

File: doc/nrf/releases/release-notes-changelog.rst (hunk 3 above)
Comment (KAGA164) [reply doc/nrf/releases/release-notes-changelog.rst]: Done

File: doc/nrf/releases/release-notes-changelog.rst (hunk 4 above)
Comment (KAGA164) [reply doc/nrf/releases/release-notes-changelog.rst]: Done

**Review by peknis — APPROVED** [review]

**Review by kapi-no — APPROVED** [review]

**Pull request merged**